### DISCUSSION TITLE
Capcom vs snk 2 naomi arcade doesn't save the scores

### DISCUSSION THREAD

**Discussion by Juniorsouza123** [flyinghead/flycast#1750] [General]
Good afternoon flying head, do you have any idea why Capcom vs snk 2 naomi arcade doesn't save scores?
the same happens with marvel vs capcom 2

**Comment by Juniorsouza123** [comment]
![010](https://github.com/user-attachments/assets/4e20e93a-b2c9-48cd-ab03-25a52be52c4a)

**Reply by flyinghead** [reply]
it looks like Capcom arcade games don't save high scores in nvram so they are reset at power off. I'm not 100% sure but several people mentioned it in various forums.
https://www.neo-geo.com/forums/index.php?threads/cps2-doesnt-save-hi-score.130855/#post-1858864
http://forum.arcadecontrols.com/index.php/topic,141386.msg1463837.html#msg1463837
